translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,48 @@
+# Terms
+
+- Message: رسالة
```

Comment:
You should put double line breaks after the terms - please preview the rendering of the document on GitHub.